CMakeLists.txt

```diff
@@ -479,6 +479,8 @@ if(NOT DEFAULT_TARGETS)
   endif()
 endif()
 
+set(ACPP_FILESYSTEM_SEARCH_OPTIONS "Final Experimental Boost" CACHE STRING "Which filesystem libraries should be searched for")
```

Comment:
The `src` subdirectory, where this variable is used, is included already above this line. Could there be problems for the first cmake invocation, when the cache is empty, since this line has not yet been executed at the point when it the variable is used?

Reply:
completely missed the include order there and the other use in `src/runtime/CMakeLists.txt`, fixing now

Reply:
another question, would it make sense to search for the filesystem library in one single place instead of doing it separately in both common and runtime?

Reply:
I might get around to that to change the code in runtime to use the methods from common, but need to work on actual job first :)

Reply:
No worries, it's not something that is urgent, and also not something that needs to be done in the context of this PR :)